pkg/controller/installation/core_controller.go

```diff
@@ -495,6 +495,26 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 	// Convert specified and detected settings into render configuration.
 	netConf := GenerateRenderConfig(instance)
 
+	if netConf.BPFEnabled {
+		var err error
+		netConf.K8sHost, netConf.K8sPort, err = utils.GetK8sEndpoint(r.client)
+		if err != nil {
+			return reconcile.Result{}, err
+		}
+	}
+
+	kubeProxyState, err := utils.GetKubeProxyState(r.client)
```

Comment:
Like we discussed in Slack, I do not think the operator should be in the business of modifying the kube-proxy (either installing it, modifying it, removing it, etc). 

Users should either install or remove the kube-proxy as needed independently of installing our operator.

Reply:
ohhh this is not supposed to be here anymore as well as the other similar, I thought that I removed it :-/ thanks for spotting this!